decidim-admin/app/views/decidim/admin/participatory_space_private_users/index.html.erb

```diff
@@ -3,7 +3,7 @@
     <h2 class="card-title">
       <%= t(".title") %>
       <% if allowed_to? :create, :space_private_user %>
-        <%= link_to t(".import_via_csv"), new_participatory_space_private_users_csv_import_path, class: "button tiny button--title import" %>
+        <%= link_to t(".import_via_csv"), new_participatory_space_private_users_csv_imports_path, class: "button tiny button--title import" %>
```

Comment:
Any reason why we're making the import plural?

Reply:
It was plural at some places and singular at others.

I'm commiting the last changes when i'm sure my tests are passing